naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingAdd.java

```diff
@@ -386,7 +386,7 @@ public String getJndiViewInstanceValue() {
             }
             ManagedReference reference = getReference();
             try {
-                return String.valueOf(reference.getInstance());
```

Comment:
This is not valid: String.valueOf(null) returns a String with the word "null". (String) null returns a null reference.